content/en/docs/Responses/error.md

```diff
@@ -42,6 +42,7 @@ The following error codes are defined:
 | 41   | Token authentication not supported for LDAP users.                                                                    |
 | 42   | Password authentication not supported. Use API keys                                                                   |
 | 43   | Multiple conflicting authentication mechanisms provided                                                               |
+| 44   | Invalid API key or username                                                                                           |
```

Comment:
I think the new apiKey version no more pass an username so it can't be a wrong user name.

Reply:
The reason why I left username is if it's not required (and clients just specify api key), then there's no easy way to get the username. I would _potentially_ be amenable to adding a new endpoint to turn a token into a username

Reply:
Ho you left username mandatory I missed that. Well then it makes sense but won't it be a problem if we extend to v2 with apiKey that can be limited to a media and don't want to leak the username in the urls ?

Reply:
Sounds like a new endpoint to exchange token for username (and other things (?) for v2) then

Reply:
Sounds good, could return the scope too ;)

I think better to reuse the apiKey for media than adding again something else that would not bring anything more.